--- a/operator/Makefile
+++ b/operator/Makefile
@@ -5,7 +5,7 @@

 .PHONY: all build build-tools crds-rs generate manifests cluster-up cluster-down image push install-trustee install clean fmt-check clippy lint test test-release release-tarball

-NAMESPACE ?= trusted-execution-clusters
+NAMESPACE ?= confidential-clusters

 KUBECTL=kubectl

@@ -18,10 +18,10 @@
 KOPIUM_VERSION ?= 0.21.3
 KOPIUM ?= $(LOCALBIN)/kopium-$(KOPIUM_VERSION)

-REGISTRY ?= quay.io/trusted-execution-clusters
+REGISTRY ?= quay.io/confidential-clusters
 TAG ?= latest
 PUSH_FLAGS ?=
-OPERATOR_IMAGE=$(REGISTRY)/trusted-cluster-operator:$(TAG)
+OPERATOR_IMAGE=$(REGISTRY)/cocl-operator:$(TAG)
 COMPUTE_PCRS_IMAGE=$(REGISTRY)/compute-pcrs:$(TAG)
 REG_SERVER_IMAGE=$(REGISTRY)/registration-server:$(TAG)
 # TODO add support for TPM AK verification, then move to a KBS with implemented verifier
@@ -72,6 +72,7 @@
 		-pcrs-compute-image $(COMPUTE_PCRS_IMAGE) \
 		-register-server-image $(REG_SERVER_IMAGE) \
 		-approved-image $(APPROVED_IMAGE)
+	../hack/customize-manifests.sh

 cluster-up:
 	RUNTIME=$(RUNTIME) scripts/create-cluster-kind.sh
@@ -102,9 +103,8 @@
 	tar -cf trusted-execution-operator-$(TAG).tar config

 # OLM Bundle related variables
-BUNDLE_DIR := bundle
-BUNDLE_IMAGE := $(REGISTRY)/trusted-cluster-operator-bundle:$(TAG)
-BUNDLE_PACKAGE ?= trusted-cluster-operator
+BUNDLE_DIR := ../bundle
+BUNDLE_IMAGE := $(REGISTRY)/cocl-operator-bundle:$(TAG)
 PREVIOUS_CSV ?= ""  # optional previous CSV for OLM upgrades

 .PHONY: bundle bundle-image push-bundle
@@ -114,7 +114,7 @@
 	@OPERATOR_IMAGE=$(OPERATOR_IMAGE) \
 	COMPUTE_PCRS_IMAGE=$(COMPUTE_PCRS_IMAGE) \
 	REG_SERVER_IMAGE=$(REG_SERVER_IMAGE) \
-	scripts/generate-bundle-prod.sh -v $(TAG) $(if $(PREVIOUS_CSV),-p $(PREVIOUS_CSV))
+	../hack/generate_bundle.sh -v $(TAG) $(if $(PREVIOUS_CSV),-p $(PREVIOUS_CSV))

 bundle-image: bundle
 	@echo "Building OLM bundle image..."
@@ -187,7 +187,7 @@
 $(KOPIUM): $(LOCALBIN)
 	$(call cargo-install-tool,$(KOPIUM),kopium,$(KOPIUM_VERSION))

-build-tools: $(CONTROLLER_GEN) $(KOPIUM)
+build-tools: $(CONTROLLER_GEN) $(KOPIUM) $(YQ)

 define go-install-tool
 [ -f "$(1)" ] || { \
